### PR TITLE
[Components with more than 1 service] LPS-192202 Refactor export-import components

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @GabrielHenriquedev, please break your changes into small commits. Thx

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

File: portal-kernel/src/com/liferay/exportimport/kernel/controller/ExportImportControllerRegistryUtil.java [hunk 1]
```diff
@@ -24,10 +21,6 @@ public static ExportController getExportController(String className) {
 		return (ExportController)_exportControllers.getService(className);
 	}
 
-	public static List<ExportImportController> getExportImportControllers() {
```
Comment (rafaprax) [inline portal-kernel/src/com/liferay/exportimport/kernel/controller/ExportImportControllerRegistryUtil.java:27]: Hey @GabrielHenriquedev, this change should be in a specific commit.

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/PortletExportControllerImpl.java [hunk 2]
```diff
@@ -113,8 +112,11 @@
  * @author Máté Thurzó
  */
 @Component(
-	property = "model.class.name=com.liferay.portal.kernel.model.Portlet",
-	service = {ExportImportController.class, PortletExportController.class}
+	property = {
+		"export.controller=true",
```
Comment (rafaprax) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/PortletExportControllerImpl.java:116]: Hey @GabrielHenriquedev, the change in the tracker that supports this modification should also be in this commit. Thx

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/PortletImportControllerImpl.java [hunk 3]
```diff
@@ -130,8 +129,11 @@
  * @author Máté Thurzó
  */
 @Component(
-	property = "model.class.name=com.liferay.portal.kernel.model.Portlet",
-	service = {ExportImportController.class, PortletImportController.class}
+	property = {
+		"import.controller=true",
+		"model.class.name=com.liferay.portal.kernel.model.Portlet"
+	},
+	service = PortletImportController.class
```
Comment (rafaprax) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/PortletImportControllerImpl.java:136]: Hey @GabrielHenriquedev, the change in the tracker that supports this modification should also be in this commit. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
See my comments and resend. Thx

**Pull request closed without merging**